projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-filtering.component.ts

```diff
@@ -116,7 +116,14 @@ export class IgxGridExcelStyleFilteringComponent extends BaseFilteringComponent
     public sortingChanged = new EventEmitter();
 
     /**
-     * @hidden @internal
+     * Emitted when the column changed.
+     *
+     * ```html
+     * <igx-grid-excel-style-filtering (columnChange)="columnChange($event)">
+     * </igx-grid-excel-style-filtering>
+     * ```
+     *
+     * @memberof IgxGridExcelStyleFilteringComponent
      */
     @Output()
     public columnChange = new EventEmitter<ColumnType>();
```

Comment:
Eh, can the EFS component change it's column on its own through interaction? Otherwise, this is hooked to the public `@Input column` setter and really doesn't provide much outside of some odd internal handling.

Reply:
yes, all right.